packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -39,6 +39,12 @@ const conf = (module.exports = convict({
       env: 'AMPLITUDE_DISABLED',
       format: Boolean,
     },
+    rawEvents: {
```

Comment:
Not related but strange that `amplitude.disabled` doesn't disable everything amplitude. Would `logRawEvents` be a more clear config value name?

Reply:
Yep, Ben also raise this at https://github.com/mozilla/fxa/pull/4718/files#r401784949